web/init/src/scss/components/kustomize/HelmValuesEditor.scss

```diff
@@ -1,4 +1,11 @@
-.AceEditor--wrapper.helm-values {
+.AceEditor--wrapper {
+  max-height: 68vh;
```

Comment:
what happens if you have a really tall display? Would there just be lots of whitespace at the bottom?

My thinking as to how to solve this was looking at the flex properties and the wrappers around it so that no matter the height of the display it would flex to all available space.